docs/docsite/end_of_life.toml

```diff
@@ -0,0 +1,25 @@
+[distribution.ansible]
+end-of-life = [
+  'stable-2.15',
+  'stable-2.14',
+  'stable-2.13',
+]
+supported = [
+  'devel',
+  'stable-2.18',
+  'stable-2.17',
+  'stable-2.16',
```

Comment:
The community package is only built for the latest ansible-core version. Should these be removed?

```suggestion
```

Reply:
Dunno, I'm yet to fully understand the context. I'd leave this to Don. Or maybe we'll get to doing something together.